WordPress/Classes/ViewRelated/Me/MeViewController.m

```diff
@@ -200,16 +201,16 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             AccountService *accountService = [[AccountService alloc] initWithManagedObjectContext:context];
             WPAccount *defaultAccount = [accountService defaultWordPressComAccount];
 
-            cell.textLabel.textAlignment = NSTextAlignmentCenter;
             cell.accessoryType = UITableViewCellAccessoryNone;
 
             if (defaultAccount) {
-                NSString *signOutString = NSLocalizedString(@"Sign Out", @"Sign out from WordPress.com");
+                NSString *signoutFromAccunt = [NSString stringWithFormat:@"Unlink %@", defaultAccount.username];
+                NSString *signOutString = NSLocalizedString(signoutFromAccunt, @"Sign out from WordPress.com");
```

Comment:
We need to localize the template, not the formatted string. So something like this: 
NSLocalizedString(@"Unlink %@", @"Label for the sign out button. The %@ is a placeholder for the user's screen name.");
The description let's the polyglots know how to localize the string so the username is in the correct place.